tests/auto/validate_prompts.test.js

```
@@ -0,0 +1,861 @@
+      fs.writeFileSync(path.join(testDir, '.claude', 'commands', 'cmd1.md'), 'content');
+      fs.writeFileSync(path.join(testDir, '.claude', 'commands', 'cmd2.md'), 'content');
+      
+      // Set environment variable expecting 3 commands
+      process.env.EXPECTED_COMMAND_COUNT = '3';
+      
+      validator.validateSystemIntegrity();
+      
+      expect(validator.errors).to.include('Expected 3 commands, found 2');
+      
+      // Clean up
+      delete process.env.EXPECTED_COMMAND_COUNT;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)